install/installer/pkg/components/ws-daemon/daemonset.go

```diff
@@ -293,6 +293,19 @@ fi
 				SecurityContext: &corev1.SecurityContext{
 					AllowPrivilegeEscalation: pointer.Bool(false),
 				},
+				LivenessProbe: &corev1.Probe{
+					ProbeHandler: corev1.ProbeHandler{
+						HTTPGet: &corev1.HTTPGetAction{
+							Path: "/ready",
+							Port: intstr.IntOrString{IntVal: ReadinessPort},
+						},
+					},
+					InitialDelaySeconds: 5,
+					PeriodSeconds:       2,
+					TimeoutSeconds:      1,
```

Comment:
Will the timeout value too aggressive if the ws-daemon is busy on backing up workspace content to remove?

Reply:
wsdaemon should not stop responding to the probe when it is backing up workspace. Backup of workspace is not blocking. 
So I would imagine this probe should only fail if wsdaemon becomes so busy (too many backup operations at once?) that it even fails to respond to the probe. In that case disabling new workspaces on it is a good idea anyway.

Reply:
I align currently ws-daemon's liveness probe, but it would be too aggressive..? WDYT?
https://github.com/gitpod-io/gitpod/pull/15262/files#diff-131eab3a970d46aaf4807ed70b6cbf2dd22bb263f2a7692ebc3dfe6a104f617fR274

Reply:
@jenting I think this may be better from the perspective of burden to ws-daemon when it running in the backup phase since we are just removing the ready label.

Reply:
I considered increasing the timeoutSeconds.
But since the backup workspace content would not block the probe response, I am okay with it now. 😃

Reply:
Exactly, @utam0k . If this liveness probe triggers, it means we'll remove the ws-daemon readiness label, and Kubernetes will not schedule workspace pods to the node, until the label is added back. In other words, it should not negatively impact workspaces that are actively initializing or backing up.